src/main/java/com/ebay/sonar/test/github/Main2.java

```diff
@@ -0,0 +1,14 @@
+package com.ebay.sonar.test.github;
+
+/**
+ * Foo
+ *
+ */
+public class Main2 {
+
+  public int simpleMethod() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AUndocumentedApi)